mappings/net/minecraft/client/gui/widget/ClickableWidget.mapping

```diff
@@ -81,8 +81,8 @@ CLASS net/minecraft/class_339 net/minecraft/client/gui/widget/ClickableWidget
 	METHOD method_75799 drawTextWithMargin (Lnet/minecraft/class_12225;Lnet/minecraft/class_2561;I)V
 		ARG 1 drawer
 		ARG 2 text
-		ARG 3 xMargin
-	CLASS class_12230 ToggleableText
-		FIELD field_63861 grayedMessage Lnet/minecraft/class_2561;
-		METHOD method_75800 makeGray (Lnet/minecraft/class_2561;)Lnet/minecraft/class_2561;
+		ARG 3 marginX
+	CLASS class_12230 InactivityIndicatingWidget
```

Comment:
This is a subclass of Widget, not Text; it is not "toggleable" (note that some ClickableWidget subclasses are - say, checkboxes), and methods should be named after purpose, not implementation detail.